recipes/json-schema-validator/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+from conans.tools import Version
+
+
+class JsonSchemaValidatorConan(ConanFile):
+    name = "json-schema-validator"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/pboettch/json-schema-validator"
+    description = "JSON schema validator for JSON for Modern C++ "
+    topics = ("json-schema-validator", "modern-json",
+              "nlohmann_json", "conan-recipe")
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt"]
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    short_paths = True
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        compiler_version = Version(self.settings.compiler.version)
+        if self.settings.os == "Windows"and self.settings.compiler == "Visual Studio" and compiler_version < "16":
+            tools.check_min_cppstd(self, "17")
+        elif self.settings.os == "Linux" and self.settings.compiler == "clang" and compiler_version < "4":
+            tools.check_min_cppstd(self, "11")
+        elif self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, "11")
+
+    def requirements(self):
+        self.requires("nlohmann_json/3.7.3")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob(self.name + "-*/")[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_TESTS"] = False
+        self._cmake.definitions["BUILD_EXAMPLES"] = False
+        self._cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
```

Comment:
```suggestion
```

Automatically configured by Conan Cmake helper

Reply:
:exploding_head: Conan is just too smart